modules/doc/content/application_development/profiling.md

```diff
@@ -7,6 +7,33 @@ installation process.  Although it works best on Linux platforms, it also
 works reasonably well on Mac OS.  Instruments also works well for profiling
 applications on Mac systems.
 
+## Install Google's gperftools
+
+If your environment package from the MOOSE installation process has included Google's gperftools,
+you can skip this step.
+To install Google's gperftools on your own, our recommended procedure is
+
+```
+cd $HOME
+git clone git@github.com:gperftools/gperftools.git
+cd gperftools
+./configure --prefix=$PWD/installed --enable-frame-pointers
```

Comment:
For me I need to do `./autogen.sh` after cloning before using this command.

Reply:
Good catch. I believe I forgot this one.

Reply:
It is correct if you are working a fresh clone. We should mention this. Thanks for pointing that out

Reply:
Added `./autogen.sh` step for `gperftools` and `autoreconf -i` step for `libunwind` in newest commit